web/beacon-app/src/features/auth/components/Register/schemas/registrationFormValidation.ts

```diff
@@ -4,7 +4,7 @@ import { NewUserAccount } from '../../../types/RegisterService';
 
 const validationSchema = Yup.object().shape({
   name: Yup.string().required('The name is required.'),
-  email: Yup.string().email().required('The email address is required.'),
+  email: Yup.string().email('Email is invalid.').required('The email address is required.'),
```

Comment:
This message matches the message that appears in the `Login` form when an email without an `@` symbol is typed into the form.